testsuite/tests/kuadrant/identical_hostnames/test_identical_hostnames_auth_on_gw_and_route_desired.py

```diff
@@ -0,0 +1,53 @@
+"""
+Tests desired behavior of using one HTTPRoute declaring the same hostname as parent Gateway related to AuthPolicy.
+https://github.com/Kuadrant/kuadrant-operator/blob/main/doc/auth.md#limitation-multiple-network-resources-with-identical-hostnames
+(see second topology mentioned there). This test should start passing once the policy-2 affects the route-b which is
+considered to be desired behavior.
+"""
+
+import pytest
+
+from testsuite.policy.authorization.auth_policy import AuthPolicy
+
+pytestmark = [pytest.mark.kuadrant_only]
+
+
+@pytest.fixture(scope="class", autouse=True)
+def authorization2(request, gateway, blame, openshift, label):
+    """2nd 'deny-all' Authorization object"""
+    auth_policy = AuthPolicy.create_instance(openshift, blame("authz2"), gateway, labels={"testRun": label})
+    auth_policy.authorization.add_opa_policy("rego", "allow = false")
+    request.addfinalizer(auth_policy.delete)
+    auth_policy.commit()
+    auth_policy.wait_for_ready()
+    return auth_policy
+
+
+@pytest.mark.xfail(
+    strict=True,
```

Comment:
I think this is not needed, we have xpassfail enabled by default

Reply:
OK, will remove this for all xfalis across identical hostname tests

Reply:
Done